frontend/test/__support__/server-mocks/database.ts

```diff
@@ -45,9 +45,11 @@ export const setupSchemaEndpoints = (db: Database) => {
 };
 
 export function setupDatabaseIdFieldsEndpoints({ id, tables = [] }: Database) {
-  const fields = tables
-    .flatMap(table => table.fields ?? [])
-    .filter(field => isTypeFK(field.semantic_type));
+  const fields = tables.flatMap(table =>
```

Comment:
tables are always hydrated in this endpoint, otherwise some machinery won't work